File: Linguagem de Programação I/Extras/Beecrowd/2709.py
def is_prime(n):
    if n < 2:
        return False
    if n == 2:
        return True
    if n % 2 == 0:
        return False
    p = 3
    while p * p <= n:
        if n % p == 0:
            return False
        p += 2
    return True

while True:
    try:
        M = int(input())
        coins = [int(input()) for _ in range(M)]
        N = int(input())
        sum_coins = sum(coins[::-N])
        if is_prime(sum_coins):
            print("You’re a coastal aircraft, Robbie, a large silver aircraft.")
        else:
            print("Bad boy! I’ll hit you.")
    except EOFError:
        break

File: Linguagem de Programação I/Extras/Beecrowd_Extra/2709.php
<?php
function isPrime($n) {
    if ($n < 2) return false;
    if ($n == 2) return true;
    if ($n % 2 == 0) return false;
    for ($p = 3; $p * $p <= $n; $p += 2) {
        if ($n % $p == 0) return false;
    }
    return true;
}

while (true) {
    $M = intval(fgets(STDIN));
    if ($M === 0) break;
    $coins = [];
    for ($i = 0; $i < $M; $i++) {
        $coins[] = intval(fgets(STDIN));
    }
    $N = intval(fgets(STDIN));
    $sumCoins = 0;
    for ($i = $M - 1; $i >= 0; $i -= $N) {
        $sumCoins += $coins[$i];
    }
    if (isPrime($sumCoins)) {
        echo "You’re a coastal aircraft, Robbie, a large silver aircraft.\n";
    } else {
        echo "Bad boy! I’ll hit you.\n";
    }
}
?>